components/automate-ui/src/app/page-components/user-table/user-table.component.ts

```diff
@@ -8,16 +7,13 @@ import { User } from 'app/entities/users/user.model';
   styleUrls: ['./user-table.component.scss']
 })
 export class UserTableComponent implements OnInit {
-  @Input() addButtonText: string;
   @Input() removeText: string;
-  @Input() addButtonEnabled = true;
-  @Input() users$: Observable<User[]>;
+  @Input() users: User[];
```

Comment:
we decided to make this component a little bit dumber by passing it a regular User array instead of an Observable User array